README.md

```diff
@@ -124,6 +124,39 @@ If you want to check what resources are going to be targeted without actually te
 cloud-nuke aws --resource-type ec2 --dry-run
 ```
 
+### Specify optional config file
+
+For more granularity, you can pass in a config file to specify which resources to terminate using regular expressions.
+
+```shell
+cloud-nuke --config path/to/file.yaml
+```
+
+For example, if you want to terminate S3 buckets whose name matches the given patterns, here's what your config file might look like:
+```yaml
+s3:
+  name_filter:
+    - ^alb-alb-.*-access-logs$
+    - .*-prod-alb-.*
+```
```

Comment:
This could use a bit more explanation. For example, is it an `and` or `or` expression? That is, does the name need to match all the regexes or only one?

Showing an example would probably help. Something like:

> Given this config, `cloud-nuke` will only nuke S3 buckets that match one of the provided regular expressions. So a bucket named `alb-alb-app-access-logs` would be deleted, but not a bucket named `my-s3-bucket`.